src/main/java/players/rmhc/RMHCPlayer.java

```diff
@@ -106,10 +111,6 @@ private void runIteration(AbstractGameState stateObs) {
         avgTimeTaken = acumTimeTaken / numIters;
     }
 
-    @Override
-    public String toString() {
```

Comment:
Why remove?

Reply:
Because I have moved name up to the abstract super class, so the behaviour is the same. (Or at least, it would have been if I hadn't accidentally removed the line that set the default name to "RMHC"...that is now being added back in . Ahem.)